.claude/hooks/python_check/file_extractor.py

```
@@ -52,19 +51,19 @@ def _extract_paths_from_payload(payload: Any) -> List[str]:
 
 def get_modified_files(
-    tool_name: str, tool_input: Dict[str, Any], tool_response: Dict[str, Any]
-) -> Tuple[List[str], List[str]]:
+    tool_name: str, tool_input: dict[str, Any], tool_response: dict[str, Any]
+) -> tuple[list[str], list[str]]:
     """Extract file paths that were modified from tool input/response."""
-    files: List[str] = []
+    files: list[str] = []
 
     if tool_name in FILE_MODIFICATION_TOOLS:
         # Standard file modification tools – collect from known shapes
```

Ruff (0.12.2)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)